src/tests/ftest/daos_test/daos_core_test-rebuild.py

```diff
@@ -32,6 +32,12 @@ class DaosCoreTestRebuild(DaosCoreBase):
     :avocado: recursive
     """
 
+    # Test variants that should be skipped
+    CANCEL_FOR_TICKET = [
+        ["DAOS-5851", "test_name", "rebuild tests 0-10"],
```

Comment:
This is missing the corresponding change to uncomment the following https://github.com/daos-stack/daos/blob/master/src/tests/ftest/daos_test/daos_core_test-rebuild.yaml#L60-L65.  Without it we will not see a CANCEL state for the Rebuild 0-10 test varaint.

Reply:
yes, let me do that in a separate PR. Let's not delay the disabling of rebuild18.